Algorithm.CSharp/Alphas/GlobalEquityMeanReversionIBSAlpha.cs

```diff
@@ -0,0 +1,149 @@
+using QuantConnect.Algorithm.Framework.Alphas;
+using QuantConnect.Algorithm.Framework.Execution;
+using QuantConnect.Algorithm.Framework.Portfolio;
+using QuantConnect.Algorithm.Framework.Risk;
+using QuantConnect.Algorithm.Framework.Selection;
+using QuantConnect.Data;
+using QuantConnect.Data.UniverseSelection;
+using QuantConnect.Orders.Fees;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuantConnect.Algorithm.CSharp.Alphas
+{
+    /// <summary>
+    /// Equity indices exhibit mean reversion in daily returns. The Internal Bar Strength indicator (IBS), 
+    /// which relates the closing price of a security to its daily range can be used to identify overbought
+    /// and oversold securities.
+
+    /// This alpha ranks 33 global equity ETFs on its IBS value the previous day and predicts for the following day
+    /// that the ETF with the highest IBS value will decrease in price, and the ETF with the lowest IBS value 
+    /// will increase in price.
+
+    /// Source: Kakushadze, Zura, and Juan Andr�s Serur. �4. Exchange-Traded Funds (ETFs).� 151 Trading Strategies, Palgrave Macmillan, 2018, pp. 90�91.
+
+    /// <br><br>This alpha is part of the Benchmark Alpha Series created by QuantConnect which are open sourced so the community and client funds can see an example of an alpha. 
+    /// You can read the source code for this alpha on Github in <a href="https://github.com/QuantConnect/Lean/blob/master/Algorithm.CSharp/Alphas/GlobalEquityMeanReversionIBSAlpha.cs">C#</a>
+    /// or <a href="https://github.com/QuantConnect/Lean/blob/master/Algorithm.Python/Alphas/GlobalEquityMeanReversionIBSAlpha.py">Python</a>.
+    /// </summary>
+    public class GlobalEquityMeanReversionIBSAlpha : QCAlgorithmFramework
+    {
+        public override void Initialize()
+        {
+            SetStartDate(2018, 1, 1);
+
+            SetCash(100000);
+
+            // Set zero transaction fees
+            SetSecurityInitializer(security => security.FeeModel = new ConstantFeeModel(0));
+
+            // Global Equity ETF tickers
+            var symbols = new[] {
+                "ECH", "EEM", "EFA", "EPHE", "EPP", "EWA", "EWC", "EWG",
+                "EWH", "EWI", "EWJ", "EWL", "EWM", "EWM", "EWO", "EWP",
+                "EWQ", "EWS", "EWT", "EWU", "EWY", "EWZ", "EZA", "FXI",
+                "GXG", "IDX", "ILF", "EWM", "QQQ", "RSX", "SPY", "THD"}
+                .Select(x => QuantConnect.Symbol.Create(x, SecurityType.Equity, Market.USA));
+
+            // Manually curated universe
+            UniverseSettings.Resolution = Resolution.Daily;
+            SetUniverseSelection(new ManualUniverseSelectionModel(symbols));
+
+            // Use GlobalEquityMeanReversionAlphaModel to establish insights
+            SetAlpha(new MeanReversionIBSAlphaModel());
+
+            // Equally weigh securities in portfolio, based on insights
+            SetPortfolioConstruction(new EqualWeightingPortfolioConstructionModel());
```

Comment:
These are for documentation purposes please add the other framework setters with default models.